### PR TITLE
Updates to aws docs, including new directconnect metrics and mention …

### PR DESCRIPTION
**Pull request by DanielLanger**
…of alb in elb doc

### CODE REVIEW

File: content/integrations/awsdirectconnect.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+kind: integration
+git_integration_title: amazon_directconnect
+doclevel: basic
+---
+
+## Overview
+
+AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
+
+Enable this integration to see in Datadog all your Direct Connect metrics
```
Comment (kmshultz) [inline content/integrations/awsdirectconnect.md]: Can we name a few examples of useful metrics here? Also, our integration overviews don't need to tell readers to enable the integration. Let's just say "This integration collects <example metrics/events/checks> from AWS Direct Connect".

File: content/integrations/awsdirectconnect.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+title: Datadog-AWS Direct Connect Integration
+integration_title: AWS Direct Connect
+kind: integration
+git_integration_title: amazon_directconnect
+doclevel: basic
+---
+
+## Overview
+
+AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
```
Comment (kmshultz) [inline content/integrations/awsdirectconnect.md]: Our integration overviews aren't in the habit of explaining what the integrated-to thing is. If a user that lands here doesn't know what AWS Direct Connect is, it's a good bet he's not going to need this integration.

File: content/integrations/awsdirectconnect.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Enable this integration to see in Datadog all your Direct Connect metrics
+
+## Setup
+### Installation
+
+If you haven't already, set up the [Amazon Web Services integration first](/integrations/aws). There are no other installation steps that need to be performed.
+
+### Configuration
+
+In the Amazon Web Services integration tile, ensure that Direct Connect is checked under metric collection.
```
Comment (kmshultz) [inline content/integrations/awsdirectconnect.md]: In the Amazon Web Services integration tile in Datadog, tick the checkbox next to **Direct Connect** under **metric collection**.

File: content/integrations/awselb.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ To start collecting ELB metrics, the only thing you need to do is to set up our
 ### Configuration
 
 1.  Enable the [Amazon Web Services integration](/integrations/aws).
-2.  Open the [AWS integration tile](https://app.datadoghq.com/account/settings#integrations/amazon_web_services) and ensure the **ELB checkbox** on the left is checked.
+2.  Open the [AWS integration tile](https://app.datadoghq.com/account/settings#integrations/amazon_web_services) and ensure the **ELB checkbox** on the left is checked for Classic ELB metrics and the **ApplicationELB** checkbox for Application ELB metrics
```
Comment (kmshultz) [inline content/integrations/awselb.md]: Is there some reason we link to the AWS integration tile here, but not in the Direct Connect instructions? Let's be consistent. :)

File: content/integrations/awsdirectconnect.md (hunk 2 above)
Comment (DanielLanger) [inline content/integrations/awsdirectconnect.md]: Do you mean I should elaborate further?

Comment (kmshultz) [inline content/integrations/awsdirectconnect.md]: Actually what I meant is we can omit this. All we need is something like "This integration collects metrics from AWS Direct Connect (e.g. <metric 1>, <metric 2>)" where metric 1 and 2 are some that users are likely to care about most.

I realize that's very austere, but it seems like it's all we need to say in this case.

File: content/integrations/awsdirectconnect.md [hunk 5]
```diff
@@ -8,9 +8,9 @@ doclevel: basic
 
 ## Overview
 
-AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
+This integration collects metrics from AWS Direct Connect (e.g. `aws.dx.connection_state`, `aws.dx.connection_bps_egress`)
```
Comment (kmshultz) [inline content/integrations/awsdirectconnect.md]: I am looking for human-readable examples here. We've got the metrics section below for fully qualified metric names, e.g. "This integrations collect metrics from AWS Direct Connect like connection states, BPS egress, and more."

File: content/integrations/awsdirectconnect.md [hunk 6]
```diff
@@ -8,9 +8,9 @@ doclevel: basic
...
 
-Enable this integration to see in Datadog all your Direct Connect metrics
+Enable this integration to see in Datadog all your Direct Connect metrics.
```
Comment (kmshultz) [inline content/integrations/awsdirectconnect.md]: Let's omit this sentence. As with any integration, it's obvious that collected metrics can be seen in Datadog.

File: content/integrations/awsdirectconnect.md (hunk 5 above)
Comment (DanielLanger) [reply content/integrations/awsdirectconnect.md]: Great call, should have realized that.

File: content/integrations/awsdirectconnect.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+title: Datadog-AWS Direct Connect Integration
+integration_title: AWS Direct Connect
+kind: integration
+git_integration_title: amazon_directconnect
+doclevel: basic
+---
+
+## Overview
+
+This integration collects metrics from AWS Direct Connect (e.g. Connection state, bit rate ingress and egress, packet rate ingress and egress, and more)
```
Comment (kmshultz) [inline content/integrations/awsdirectconnect.md]: The sentence needs a period. Then 🚢 !

**Pull request merged**